src/pages/server/login.vue

```diff
@@ -123,3 +129,10 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style scoped>
+.server-used {
+  text-align: center;
+  opacity: 0.6;
+}
+</style>
```

Comment:
It can now be removed

Reply:
We usually also have Vuetify classes for this, so check them out first: https://vuetifyjs.com/en/styles/text-and-typography/